shared/constants/app.ts

```diff
@@ -60,10 +60,15 @@ export const MESSAGE_TYPE = {
   SNAP_DIALOG_CONFIRMATION: DIALOG_APPROVAL_TYPES.confirmation,
   SNAP_DIALOG_PROMPT: DIALOG_APPROVAL_TYPES.prompt,
   SNAP_DIALOG_DEFAULT: DIALOG_APPROVAL_TYPES.default,
+  HYPERLIQUID_REFERRAL_CONSENT: 'hyperliquid_referral_consent',
 } as const;
 
 export type MessageType = (typeof MESSAGE_TYPE)[keyof typeof MESSAGE_TYPE];
 
+// Custom ApprovalType for Hyperliquid referral consent
+export const HYPERLIQUID_APPROVAL_TYPE =
```

Comment:
Minor, ideally, these are defined centrally in `controller-utils` but that's a lot of overhead.

The next best thing would probably be a client equivalent enum such as `ApprovalType` in here so all these similar constants are defined together.

Reply:
I'm leaving this for now and will make adding it to the `controller-utils` package a follow up.